tensorrt_llm/serve/openai_server.py

```
@@ -1441,6 +1447,95 @@ async def create_mm_embedding_response(promise: RequestOutput):
+                usage=UsageInfo(
+                    prompt_tokens=total_prompt_tokens,
+                    total_tokens=total_prompt_tokens,
+                    completion_tokens=0,
+                ),
+            )
+            return JSONResponse(content=response.model_dump())
+
+        except CppExecutorError:
+            logger.error(traceback.format_exc())
+            signal.raise_signal(signal.SIGINT)
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)